src/_includes/landing/sponsors.njk

```diff
@@ -1,16 +1,19 @@
 {% from "components/button.njk" import button %}
 
-<section class="bg-blue-600 text-white relative px-4 sm:px-20 md:px-40 pt-10" id="sponsors">
-    <div class="mx-auto py-16">
+<section class="bg-blue-600 text-white relative px-4 md:px-8 lg:px-16 pt-10" id="sponsors">
+    <div class="container mx-auto py-16">
         <!-- Desktop image visible -->
         <div class="grid grid-cols-1 md:grid-cols-2 gap-4 sm:gap-10">
-            <div>
-                <h2 class="text-4xl font-extrabold mb-4">Event Sponsors</h2>
-                <p class="text-white text-base mb-6">
+            <div class="flex flex-col gap-9">
+                <h2 class="text-4xl text-white font-extrabold font-['Black_Han_Sans','sans-serif']">Event Sponsors</h2>
```

Comment:
why is this font included?

Reply:
I am wondering if it has any effect on the websit 🤔

Reply:
If you check https://in.pycon.org/2025/ the Sponsors and Community section has different kinds of font

Reply:
good catch, can you make sure they have the same font and required size as show in figma?